Makefile.PL

```diff
@@ -111,18 +111,16 @@ my %eumm_args = (
   ssleay(),
 );
 
-# CCFLAGS is used internally by Makefile.PL to define various C preprocessor
-# macros (as opposed to DEFINE, which is user-facing).
-$eumm_args{CCFLAGS} = $Config{ccflags};
```

Comment:
`SSLeay.xs` currently says:
https://github.com/radiator-software/p5-net-ssleay/blob/420b801fa36370b4043803d34c65b1d334958318/SSLeay.xs#L209-L214

It would be good to leave `DEFINE` for command line use, as shown above, but I couldn't find an easy way to do this. In other words, passing defines to the compiler directly from `Makefile.PL` *and* also allowing additional defines defined on the command line. Would you happen to know a way to do this?

Reply:
I see what you mean.  The MakeMaker docs seem pretty clear that you can send an attribute to WriteMakefile _or_ you can make it an argument on the command line -- not both. The only alternative I can think of would be to parse off any DEFINEs found in @ARGV and add the concatenated result to the DEFINE that gets passed to WriteMakefile.  That's not pretty but I can't think of anything better at the moment.  Maybe inspiration will strike.